types/aws-lambda/trigger/lex-v2.d.ts

```diff
@@ -74,38 +68,46 @@ export interface LexV2SentimentScore {
 
 export interface LexV2SessionState {
     activeContexts?: LexV2ActiveContext[];
-    sessionAttributes?: { [key: string]: string };
+    sessionAttributes?: Record<string, string>;
     dialogAction?: LexV2DialogAction;
     intent: LexV2Intent;
     originatingRequestId: string;
 }
 
 export interface LexV2ActiveContext {
     name: string;
-    contextAttributes: { [key: string]: string };
+    contextAttributes: Record<string, string>;
     timeToLive: {
         timeToLiveInSeconds: number;
         turnsToLive: number
     };
 }
 
-export interface LexV2DialogAction {
-    slotToElicit?: string;
-    type: 'Close' | 'ConfirmIntent' | 'Delegate' | 'ElicitIntent' | 'ElicitSlot';
-}
+export type LevV2DialogActionWithoutSlot =
```

Comment:
Probably shouldn't export machinery types, because you can't rename them later, but pretty minor